kuma/users/middleware.py

```diff
@@ -17,8 +17,10 @@ def process_request(self, request):
             if not bans:
                 return None
             logout(request)
-            return render(request,
-                          'users/user_banned.html',
-                          {'bans': bans,
-                           'path': request.path})
+            banned_response = render(request, 'users/user_banned.html', {
+                'bans': bans,
+                'path': request.path
+            })
+            banned_response['Cache-Control'] = 'private, no-cache'
+            return banned_response
```

Comment:
I'm not sure I like `private`. It suggests it's ok to cache this response in some situations (if the cache is private to this user).

Instead I would follow Django's `@never_cache` decorator and do what it does:
https://github.com/django/django/blob/master/django/views/decorators/cache.py#L57-L59